### DISCUSSION TITLE
Double-click to edit card

### DISCUSSION THREAD

**Discussion by tf13** [mgmeyers/obsidian-kanban#34] [Ideas & Feature Requests]
Double-clicking the card text area should allow editing (as opposed to having to click the three-dots menu).

**Comment by mgmeyers** [comment]
I believe you've already seen @tf13, but in case anyone else hasn't yet: you can now cmd/ctrl + click on a card's text to edit it.

**Comment by mgmeyers** [comment]
FYI: I've decided to go with double-clicking as requested, rather than cmd/ctrl + click.
